easybuild/easyconfigs/l/LAST/LAST-869-GCCcore-6.3.0.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'MakeCp'
+
+name = 'LAST'
+version = '869'
+
+homepage = 'http://last.cbrc.jp/'
+description = """ LAST finds similar regions between sequences. The main technical innovation
+ is that LAST finds initial matches based on their multiplicity,
+ instead of using a fixed length (e.g. BLAST uses 11-mers)."""
+
+toolchain = {'name': 'GCCcore', 'version': '6.3.0'}
```

Comment:
hmm, wouldn't `intel` make more sense here, for performance reasons?

Reply:
@boegel Good question, I am afraid of `intel` compiler in biostuff.